cli/tests/subdir/test_worker_basic.js

```diff
@@ -6,12 +6,10 @@ if (self.name !== "jsWorker") {
 }
 
 onmessage = function(e) {
-  console.log("jsWorker onmessage", e.data);
   postMessage(e.data);
   close();
 };
 
 onerror = function() {
-  console.log("called onerror in worker");
   return false;
 };
```

Comment:
I don’t understand why all these console.log statements are being removed?

Reply:
Not needed and was harder to match output in integration test